js/ui/overlay/utils.js

```diff
@@ -11,4 +14,33 @@ const getElementMaxHeightByWindow = $element => {
     return Math.max(offsetTop, offsetBottom) * WINDOW_HEIGHT_PERCENT;
 };
 
-export { getElementMaxHeightByWindow };
+const baseZIndex = (ZIndex) => {
+    if(!ZIndex) {
+        return BASE_ZINDEX;
+    }
+    BASE_ZINDEX = ZIndex;
+};
+
+const createNewZIndex = (baseIndex) => {
+    const length = ZIndexStack.length;
+    const index = length ? ZIndexStack[length - 1] + 1 : (baseIndex || baseZIndex()) + 1;
+
+    ZIndexStack.push(index);
+
+    return index;
+};
+
+const removeZIndex = (zIndex) => {
+    ZIndexStack.forEach(function(index, position) {
+        if(index === zIndex) {
+            ZIndexStack.splice(position, 1);
+            return false;
```

Comment:
`return false;` - What does it mean?

Reply:
`return false` is used to stop the loop: [https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#Example:_Breaking_a_loop](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#Example:_Breaking_a_loop)